src/certifier.cc

```diff
@@ -276,15 +276,15 @@ uint64_t get_tcb_version_from_vcek(X509 *vcek) {
   return tcb_version;
 }
 
-bool get_chipid_from_vcek(X509 *vcek, unsigned char *chipid, int idlen) {
+static bool get_chipid_from_vcek(X509 *vcek, unsigned char *chipid, int idlen) {
   int                  nid = -1, idx = -1, extlen = -1;
   X509_EXTENSION *     ex = NULL;
   ASN1_STRING *        extvalue = NULL;
   const unsigned char *vals = NULL;
 
   nid = OBJ_create(VCEK_EXT_HWID, VCEK_EXT_HWID, VCEK_EXT_HWID);
   if (nid == NID_undef) {
-    printf("%s() error, line %d, Failed to create NID\n", __func__, __LINE__);
+    printf("%s() Warning, line %d, Failed to create NID\n", __func__, __LINE__);
```

Comment:
We don't have a proper protocol for message severity levels. 

For now, as this message isn't really an 'error' in all cases, and as the single-caller seems to cope with a `return false`, I changed this from `error` to `Warning`.

@yelvmw -- Please advise, if this is ok, or should I just leave this as `error`.

Reply:
Discussed with @yelvmw : Get rid of message on L287; silently return `false`.